deploy/helm/sumologic/templates/deployment.yaml

```diff
@@ -47,13 +47,12 @@ spec:
           containerPort: 24321
           protocol: TCP
         livenessProbe:
-          exec:
-            command:
-            - "/bin/sh"
-            - "-c"
-            - "[ $(pgrep ruby | wc -l) -gt 0 ]"
+          httpGet:
+            path: /fluentd.pod.healthcheck?json=%7B%22log%22%3A+%22health+check%22%7D
+            port: 9880
           initialDelaySeconds: 300
-          periodSeconds: 20
+          periodSeconds: 30
+          timeoutSeconds: 3
         readinessProbe:
```

Comment:
do we want to change the `readinessProbe` as well?

Reply:
Yes, we can change the readinessProbe to be the same as well.

Reply:
did you want to make this change